### ISSUE TITLE
[HL2: EP1] Alyx's relationship with bullseye_hospitalexit (ep1_c17_05)

### ISSUE THREAD

**Issue by BreakinBenny** [ValveSoftware/Source-1-Games#3760]
If you choose to climb over the boards blocking your and Alyx's exit from the hospital, she shoots at a Bullseye entity meant to destroy those. However, breaking them yourself makes the unkilled trigger brush try to activate that. Related entities: **ptemplate_hospitalexit**, **bullseye_hospitalexit**, and an unnamed **trigger_once**.

![ep1_c17_050000](https://user-images.githubusercontent.com/4566768/146398332-6e60ea8a-46c4-4527-b147-8308a8049a61.jpg)

**Comment by Pinsplash** [comment]
This is a completely typical thing and has no conceivable effect on anything. Alyx functions exactly as intended in this area.

**Comment by BreakinBenny** [comment]
> 

Yes @Pinsplash, she does. Though maybe that brush trigger could've been killed as well if all boards are broken? It has no targetname though, but the **counter_boards_hospitalexit** would've been able to kill that easily if it did,.. but it would be better to give that brush entity an output for the math_counter to kill the trigger_once (and its hierachy?) OnHitMax.
`
"replace_entity"
{
"origin" "10068.9 12125 -598.553"
"max" "2"
"targetname" "counter_boards_hospitalexit"
"classname" "math_counter"
"OnHitMax" "bullseye_hospitalexit,Kill,,0,-1"
"OnHitMax" "brush_clip_hospitalexit,Disable,,0,-1"
"OnHitMax" "ptemplate_hospitalexit,Kill,,0,-1"
"OnHitMax" "trigger_hospitalexit,Kill,,0,-1"
}
"replace_entity"
{
"model" "*59"
"StartDisabled" "0"
"spawnflags" "1"
"targetname" "trigger_hospitalexit"
"origin" "10124 12234.9 -594.9"
"classname" "trigger_once"
"hammerid" "1646649"
"OnTrigger" "alyx,SetRelationship,bullseye_hospitalexit d_ht 100,0.1,-1"
"OnTrigger" "ptemplate_hospitalexit,ForceSpawn,,0,-1"
}
`

…If you have ideas on how to kill the trigger_once with "model *59" without giving it a targetname, go for it!

**Comment by Pinsplash** [comment]
What is the purpose in killing the trigger?

**Comment by BreakinBenny** [comment]
> @Pinsplash: What is the purpose in killing the trigger?
To prevent the console error "Couldn't set relationship to unknown entity or class!" from appearing.

**Comment by BreakinBenny** [comment]
> @Pinsplash: What is the purpose in killing the trigger?

To prevent the console error "Couldn't set relationship to unknown entity or class!" from appearing.
